pkg/platform/abstract/platform.go

```diff
@@ -1218,3 +1223,32 @@ func (ap *Platform) transformProjectDisplayNameToName(projectConfig *platform.Pr
 	// clean up display name
 	projectConfig.Spec.DisplayName = ""
 }
+
+// to sanitize potential malicious fields we focus on string fields
+func (ap *Platform) validateFunctionConfigForMaliciousInput(functionConfig *functionconfig.Config) error {
```

Comment:
🧑‍💻 
perhaps generalize it with "malformed input" - it isnt necessarily have to be malicious